lib/ProductOpener/Display.pm

```diff
@@ -5487,6 +5557,7 @@ $Lang{android_apk_app_badge}{$lc}
 			<li><a href="$Lang{footer_press_link}{$lc}">$Lang{footer_press}{$lc}</a></li>
 			<li><a href="$Lang{footer_wiki_link}{$lc}">$Lang{footer_wiki}{$lc}</a></li>
 			<li><a href="$Lang{footer_translators_link}{$lc}">$Lang{footer_translators}{$lc}</a></li>
+			<li><a href="$Lang{footer_partners_link}{$lc}">$Lang{footer_partners}{$lc}</a></li>
```

Comment:
Unrelated?

Reply:
Sorry yes those changes are unrelated, I had to push things fast in production last week-end and I took shortcuts.